packages/react-scripts/config/webpack.config.js

```diff
@@ -65,6 +65,9 @@ const sassModuleRegex = /\.module\.(scss|sass)$/;
 module.exports = function(webpackEnv) {
   const isEnvDevelopment = webpackEnv === 'development';
   const isEnvProduction = webpackEnv === 'production';
+  // Check to that is used for enabling profiling in alias object
+  const isEnvProductionProfile =
+    isEnvProduction && Boolean(process.env.PROFILE_APP);
```

Comment:
`Boolean(process.env.PROFILE_APP)` is true if `process.env.PROFILE_APP` is anything else than undefined or null (or 0 or literal `false`, which it can't be as the process.env-variables are always strings, right?) so a check like `process.env.PROFILE_APP === 'true'` makes more sense here.

Reply:
I will certainly change that 😆. I thought PROFILE_APP was primitive/literal `false` 

Thank you for pointing it out and recommending a solution. 😁

Reply:
resolved by 56d0841